node_modules/oae-zencoder/lib/init.js

```diff
@@ -13,28 +13,31 @@
  * permissions and limitations under the License.
  */
 
-var log = require('oae-logger').logger('oae-mediacore-init');
-
-var MediaCoreProcessor = require('./processor');
-
 var Cassandra = require('oae-util/lib/cassandra');
+var log = require('oae-logger').logger('oae-zencoder-init');
 var PreviewProcessorAPI = require('oae-preview-processor');
 
+var ZencoderProcessor = require('./processor');
+var ZencoderUtil = require('./util');
+
 module.exports = function(config, callback) {
-    // Register the mediacore video processor
-    PreviewProcessorAPI.registerProcessor('oae-file-mediacore', MediaCoreProcessor);
+
+    ZencoderUtil.init(config.zencoder);
+
+    // Register the zencoder video processor
+    PreviewProcessorAPI.registerProcessor('oae-file-zencoder', ZencoderProcessor);
     _ensureSchema(callback);
 };
 
 /**
- * Create the schema for the MediaCore preview processor
+ * Create the schema for the Zencoder preview processor
  *
  * @param  {Function}   callback        Standard callback function
  * @param  {Object}     callback.err    An error that occurred, if any
  * @api private
  */
 var _ensureSchema = function(callback) {
     Cassandra.createColumnFamilies({
-        'MediaCoreContentRevisionIdMap': 'CREATE TABLE "MediaCoreContentRevisionIdMap" ("mediaCoreId" text PRIMARY KEY, "contentId" text, "revisionId" text)'
+        'ZencoderToContentIdMap': 'CREATE TABLE "ZencoderToContentIdMap" ("zencoderId" text PRIMARY KEY, "contentId" text, "revisionId" text)'
```

Comment:
I think we should be able to use the admin interface to reprocess specific mime-types? As far as I've understood, it'll overwrite the mediacore metadata, We'll probably need a script to remove the mediacore tables and configuration as well when all the code has been taken out?